examples/midsquare/MidSquareHash.juvix

```diff
@@ -0,0 +1,91 @@
+-- This file implements the mid-square hashing function in Juvix. See:
```

Comment:
We could use judoc syntax to document the module. That is, use `---` instead of `--`

Reply:
done